2023-09-14 08:58:58,155 INFO    MainThread:819 [wandb_setup.py:_flush():76] Current SDK version is 0.15.9
2023-09-14 08:58:58,156 INFO    MainThread:819 [wandb_setup.py:_flush():76] Configure stats pid to 819
2023-09-14 08:58:58,156 INFO    MainThread:819 [wandb_setup.py:_flush():76] Loading settings from /home/chengp1/.config/wandb/settings
2023-09-14 08:58:58,156 INFO    MainThread:819 [wandb_setup.py:_flush():76] Loading settings from /notebooks/rl2023/ex1/wandb/settings
2023-09-14 08:58:58,156 INFO    MainThread:819 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-09-14 08:58:58,156 INFO    MainThread:819 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-09-14 08:58:58,156 INFO    MainThread:819 [wandb_init.py:_log_setup():524] Logging user logs to /notebooks/rl2023/ex1/wandb/run-20230914_085858-vkkhv3x2/logs/debug.log
2023-09-14 08:58:58,156 INFO    MainThread:819 [wandb_init.py:_log_setup():525] Logging internal logs to /notebooks/rl2023/ex1/wandb/run-20230914_085858-vkkhv3x2/logs/debug-internal.log
2023-09-14 08:58:58,157 INFO    MainThread:819 [wandb_init.py:_jupyter_setup():470] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7fc318572ef0>
2023-09-14 08:58:58,157 INFO    MainThread:819 [wandb_init.py:init():564] calling init triggers
2023-09-14 08:58:58,157 INFO    MainThread:819 [wandb_init.py:init():571] wandb.init called with sweep_config: {}
config: {'entries': {'exp_name': 'ex1', 'seed': 9, 'env_name': 'CartPole-v1', 'max_episode_steps': 100, 'train_episodes': 500, 'batch_size': 64, 'min_update_samples': 2000, 'testing': False, 'model_path': 'default', 'save_video': True, 'save_model': True, 'save_logging': True, 'silent': False, 'use_wandb': True, 'run_suffix': 0}, 'exp_name': 'ex1', 'seed': 9, 'env_name': 'CartPole-v1', 'max_episode_steps': 100, 'train_episodes': 500, 'batch_size': 64, 'min_update_samples': 2000, 'testing': False, 'model_path': 'default', 'save_video': True, 'save_model': True, 'save_logging': True, 'silent': False, 'use_wandb': True, 'run_suffix': 0}
2023-09-14 08:58:58,157 INFO    MainThread:819 [wandb_init.py:init():588] re-initializing run, found existing run on stack: tede0wyu
2023-09-14 08:58:58,160 INFO    MainThread:819 [wandb_run.py:_finish():1894] finishing run r-l/rl_aalto/tede0wyu
2023-09-14 08:58:58,160 INFO    MainThread:819 [wandb_run.py:_atexit_cleanup():2128] got exitcode: 0
2023-09-14 08:58:58,161 INFO    MainThread:819 [wandb_run.py:_restore():2111] restore
2023-09-14 08:58:58,161 INFO    MainThread:819 [wandb_run.py:_restore():2117] restore done
2023-09-14 08:59:02,581 INFO    MainThread:819 [wandb_run.py:_footer_history_summary_info():3481] rendering history
2023-09-14 08:59:02,583 INFO    MainThread:819 [wandb_run.py:_footer_history_summary_info():3513] rendering summary
2023-09-14 08:59:02,592 INFO    MainThread:819 [wandb_run.py:_footer_sync_info():3440] logging synced files
2023-09-14 08:59:02,636 INFO    MainThread:819 [wandb_init.py:init():613] starting backend
2023-09-14 08:59:02,636 INFO    MainThread:819 [wandb_init.py:init():617] setting up manager
2023-09-14 08:59:02,639 INFO    MainThread:819 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2023-09-14 08:59:02,639 INFO    MainThread:819 [wandb_init.py:init():623] backend started and connected
2023-09-14 08:59:02,650 INFO    MainThread:819 [wandb_run.py:_label_probe_notebook():1234] probe notebook
2023-09-14 08:59:02,650 INFO    MainThread:819 [wandb_run.py:_label_probe_notebook():1244] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-09-14 08:59:02,650 INFO    MainThread:819 [wandb_init.py:init():714] updated telemetry
2023-09-14 08:59:02,651 INFO    MainThread:819 [wandb_init.py:init():747] communicating run to backend with 60.0 second timeout
2023-09-14 08:59:04,099 INFO    MainThread:819 [wandb_run.py:_on_init():2180] communicating current version
2023-09-14 08:59:04,146 INFO    MainThread:819 [wandb_run.py:_on_init():2189] got version response upgrade_message: "wandb version 0.15.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2023-09-14 08:59:04,147 INFO    MainThread:819 [wandb_init.py:init():798] starting run threads in backend
2023-09-14 08:59:34,184 INFO    MainThread:819 [wandb_run.py:_console_start():2159] atexit reg
2023-09-14 08:59:34,185 INFO    MainThread:819 [wandb_run.py:_redirect():2014] redirect: wrap_raw
2023-09-14 08:59:34,185 INFO    MainThread:819 [wandb_run.py:_redirect():2079] Wrapping output streams.
2023-09-14 08:59:34,185 INFO    MainThread:819 [wandb_run.py:_redirect():2104] Redirects installed.
2023-09-14 08:59:34,186 INFO    MainThread:819 [wandb_init.py:init():839] run started, returning control to user process
2023-09-14 09:02:52,391 INFO    MainThread:819 [wandb_setup.py:_flush():76] Current SDK version is 0.15.9
2023-09-14 09:02:52,393 INFO    MainThread:819 [wandb_setup.py:_flush():76] Configure stats pid to 819
2023-09-14 09:02:52,393 INFO    MainThread:819 [wandb_setup.py:_flush():76] Loading settings from /home/chengp1/.config/wandb/settings
2023-09-14 09:02:52,393 INFO    MainThread:819 [wandb_setup.py:_flush():76] Loading settings from /notebooks/rl2023/ex1/wandb/settings
2023-09-14 09:02:52,393 INFO    MainThread:819 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-09-14 09:02:52,393 INFO    MainThread:819 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-09-14 09:02:52,393 INFO    MainThread:819 [wandb_init.py:_log_setup():524] Logging user logs to /notebooks/rl2023/ex1/wandb/run-20230914_090252-98myc4kd/logs/debug.log
2023-09-14 09:02:52,393 INFO    MainThread:819 [wandb_init.py:_log_setup():525] Logging internal logs to /notebooks/rl2023/ex1/wandb/run-20230914_090252-98myc4kd/logs/debug-internal.log
2023-09-14 09:02:52,393 INFO    MainThread:819 [wandb_init.py:init():564] calling init triggers
2023-09-14 09:02:52,394 INFO    MainThread:819 [wandb_init.py:init():571] wandb.init called with sweep_config: {}
config: {'entries': {'exp_name': 'ex1', 'seed': 66, 'env_name': 'CartPole-v1', 'max_episode_steps': 100, 'train_episodes': 500, 'batch_size': 64, 'min_update_samples': 2000, 'testing': False, 'model_path': 'default', 'save_video': True, 'save_model': True, 'save_logging': True, 'silent': False, 'use_wandb': True, 'run_suffix': 0}, 'exp_name': 'ex1', 'seed': 66, 'env_name': 'CartPole-v1', 'max_episode_steps': 100, 'train_episodes': 500, 'batch_size': 64, 'min_update_samples': 2000, 'testing': False, 'model_path': 'default', 'save_video': True, 'save_model': True, 'save_logging': True, 'silent': False, 'use_wandb': True, 'run_suffix': 0}
2023-09-14 09:02:52,394 INFO    MainThread:819 [wandb_init.py:init():588] re-initializing run, found existing run on stack: vkkhv3x2
2023-09-14 09:02:52,397 INFO    MainThread:819 [wandb_run.py:_finish():1894] finishing run r-l/rl_aalto/vkkhv3x2
2023-09-14 09:02:52,397 INFO    MainThread:819 [jupyter.py:save_history():445] not saving jupyter history
2023-09-14 09:02:52,398 INFO    MainThread:819 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2023-09-14 09:02:52,398 INFO    MainThread:819 [wandb_init.py:_jupyter_teardown():452] cleaning up jupyter logic
2023-09-14 09:02:52,398 INFO    MainThread:819 [wandb_run.py:_atexit_cleanup():2128] got exitcode: 0
2023-09-14 09:02:52,398 INFO    MainThread:819 [wandb_run.py:_restore():2111] restore
2023-09-14 09:02:52,398 INFO    MainThread:819 [wandb_run.py:_restore():2117] restore done
2023-09-14 09:02:57,047 INFO    MainThread:819 [wandb_run.py:_footer_history_summary_info():3481] rendering history
2023-09-14 09:02:57,048 INFO    MainThread:819 [wandb_run.py:_footer_history_summary_info():3513] rendering summary
2023-09-14 09:02:57,096 INFO    MainThread:819 [wandb_run.py:_footer_sync_info():3440] logging synced files
